src/mumble/Settings.h

```diff
@@ -186,6 +188,15 @@ struct OverlaySettings {
 	friend bool operator!=(const OverlaySettings &lhs, const OverlaySettings &rhs);
 };
 
+struct Profiles {
+	static const std::string s_default_profile_name;
+	static const int s_current_settings_version;
+
+	int settings_version                      = s_current_settings_version;
+	std::string activeProfileName             = s_default_profile_name;
+	QMap< std::string, Settings > allProfiles = {};
```

Comment:
Would using a `std::map` be feasible or do you have specific reasons to choose `QMap` instead?

Reply:
No will swap it for a ``std::map`` when I have the time for another refactor